packages/guideline-blocks-settings/package.json

```
@@ -37,7 +37,7 @@
     },
     "dependencies": {
         "@frontify/app-bridge": "^3.0.0-beta.66",
-        "@frontify/sidebar-settings": "^0.3.4"
+        "@frontify/sidebar-settings": "^0.3.5"
```

```suggestion
        "@frontify/sidebar-settings": "workspace:^"
```
Should help to automatically link during the release

---

Done